TRANSLATE.rst

```diff
@@ -0,0 +1,52 @@
+******************
+Translation Guide 
+******************
+
+.. contents:: 
+
+You can help to translate the Read the Docs Sphinx Theme.
+
+Languages availables:
+
+- Spanish contributed by Leonardo J. Caballero G.
+- Your Language by YOUR NAME.
+
+Installation
+============
+
+For translate the Read the Docs Sphinx Theme you need install the following packages:
```

Comment:
* `For translate` -> `For translating`
* `you need install` -> `you will need to install`